redisinsight/ui/src/pages/home/components/rdi-header/styles.module.scss

```diff
@@ -0,0 +1,32 @@
+@import "@elastic/eui/src/global_styling/index";
+
+.searchContainer {
+  max-width: 320px;
+}
+
+.containerDl {
+  padding: 0 18px;
+  @include euiBreakpoint("m", "l", "xl") {
+    padding: 0 12px;
+  }
+}
+
+.contentDL {
```

Comment:
Maybe better use global styles? (do not duplicate css) You can look example in triggered-functions page

Reply:
I may be misunderstanding, but are you saying that we should have global `css` nested somewhere within a component that has the potential to affect other component styles inadvertently?

I am all for reusing the `css` but this feels like a short term solution that could cause problems down the line.